ui/components/Game.jsx

```diff
@@ -68,13 +69,22 @@ export default class Game extends React.Component {
 
     if (!game) {
       if (player.readyAt) {
-        const TheLobby = Lobby || DelayedGameLobby;
+        const gameLobbyProps = {
+          gameLobby,
+          game: gameLobby,
+          treatment,
+          player
+        };
+
         return (
-          <TheLobby
-            gameLobby={gameLobby}
-            treatment={treatment}
-            player={player}
-          />
+          <GameLobbyContainer {...gameLobbyProps}>
+            {Lobby ? (
+              <Lobby {...gameLobbyProps} />
```

Comment:
GameLobbyContainer is already passing all the props downwards, no need to repeat the props here in Lobby/DelayedGameLobby.